app/views/claims/_claimant.html.slim

```diff
@@ -35,14 +35,22 @@ fieldset
   = f.input :address_country, collection: ClaimantForm::COUNTRIES, include_blank: false
   = f.input :address_telephone_number, as: :tel, input_html: { class: 'medium-input' }
   = f.input :mobile_number, as: :tel, input_html: { class: 'medium-input' }
-  = f.input :email_address
+
+  = f.input :contact_preference,
```

Comment:
The fieldset is definitely missing from around the contact preferences.  Github is not showing the change very well, but if you check in the develop branch yourself, you will see that the contact_preference field does have a fieldset around it ?